### PR TITLE
#158254308 Refactor asset import files

### PR DESCRIPTION
**Pull request by JaphethCA**
## What does this PR do?
- Refactors asset import script files

### Description of tasks to be completed
- move the files into a `utils` directory
- refactor import scripts to a single script file

### How can this PR be manually tested
- checkout branch and pull latest changes
- run `python utils/seed_asset.py`

## What are the relevant pivotal tracker stories
[#158254308](https://www.pivotaltracker.com/story/show/158254308)

### Screenshots
screen shot 2018-06-20 at 2 52 19 pm
screen shot 2018-06-20 at 2 52 36 pm

### CODE REVIEW

File: utils/seed_asset.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    post_asset_subcategory
+)  # noqa
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "settings")
+django.setup()
+
+
+if __name__ == '__main__':
+    filename = input('Enter csv filename (without .csv): ').strip()
+    filepath = os.path.abspath(os.path.join(os.path.dirname(__file__))) + '/'  # noqa
```
Comment (collinmutembei) [inline utils/seed_asset.py]: The input file ought to be loaded from the root of the project. Isn't that the case @egichuri ?

**Comment by collinmutembei** [comment]
Make `utils/assets/post_scripts.py`  **DRY**er by having a method in [`utils/helpers.py`](https://github.com/AndelaOSP/art-backend/blob/b7906a19cb24d194ef77bd7c8db9e2f4155a03fa/utils/helpers.py) with the recurring statements 👇 https://github.com/AndelaOSP/art-backend/blob/b7906a19cb24d194ef77bd7c8db9e2f4155a03fa/utils/asset/post_scripts.py#L28-L32

https://github.com/AndelaOSP/art-backend/blob/b7906a19cb24d194ef77bd7c8db9e2f4155a03fa/utils/asset/post_scripts.py#L87-L91

File: utils/helpers.py [hunk 2]
```diff
@@ -41,3 +41,30 @@ def display_skipped(result):
         for key, value in result.items():
 
             print('{0}\t \t{1} \t{2}'.format(value[1], key, value[0]))
+
+
+class DependencyChecker():
```
Comment (collinmutembei) [inline utils/helpers.py]: Dead code?

File: utils/seed_asset.py (hunk 1 above)
Comment (egichuri) [reply utils/seed_asset.py]: i think we can have it in the utils folder since we'll probably only use it once

Comment (collinmutembei) [reply utils/seed_asset.py]: Sounds good!

File: utils/helpers.py [hunk 3]
```diff
@@ -41,3 +41,30 @@ def display_skipped(result):
...
+
+    @classmethod
+    def has_dep(cls, column, columns):
+        instance = cls(columns)
+        return instance.has_dep(column)
+
+    # def has_dep(self, column):
+    #     if self.columns[0] == column:
+    #         return False
+    #     return True
```
Comment (egichuri) [inline utils/helpers.py]: the commented out code can be deleted

**Pull request merged**